### PR TITLE
Add Indonesia Language

### PR DESCRIPTION
**Pull request by ppabcd**
Hello,
I really like this module in Transformice. When i see the list of languages, i didn't see my country language. So, I created this one.

### CODE REVIEW

File: source/translations/id.lua [hunk 1]
```diff
@@ -0,0 +1,741 @@
+lang.en = {
```
Comment (antiafk) [inline source/translations/id.lua:1]: You forgot to edit the "en".

Comment (ppabcd) [reply source/translations/id.lua]: all done

**Comment by Chrystiang** [comment]
Thank you! Will be added in V3.2.0

**Comment by Chrystiang** [comment]
Whats your nickname in Transformice? I need it to give credits

**Comment by ppabcd** [comment]
My nickname is Ppabcd#0000

**Pull request merged**